scripts/compile-charts.js

```diff
@@ -84,4 +76,17 @@ function compileChartsBundle() {
   console.log(chalk.green('✔ Finished chart theme module'));
 }
 
+function webpackCompile(outputFilename) {
+  execSync(
+    `webpack ${path.join(__dirname, '../src/themes/charts/themes.ts')} \
+        -o dist \
+        --config=src/webpack.config.js \
+        --env filename=${outputFilename} \
+        --env library-target=commonjs`,
```

Comment:
These are no longer directly configurable from the cli after the update. I've updated these to get passed into the webpack config.

Reply:
nit: Anything you see around `/* OUI -> EUI Aliases */` was semi-automatically added when we were adding aliases and are meant to be semi-automatically removable. Pulling this into a util function, while the originals code blocks will be removed with the next major release, the util will be left over.

Reply:
Gotcha. The util is also used for the non-aliased build.